.gitignore

```diff
@@ -8,3 +8,6 @@ ent*
 tb*
 !*.vhd
 !*.vhdl
+doc/_build/*
```

Comment:
I think that `*` is not required. Also, maybe `/doc/_build` to ensure that only one dir is filtered.